src/platform/packages/shared/kbn-esql-ast/src/visitor/global_visitor_context.ts

```diff
@@ -382,7 +391,18 @@ export class GlobalVisitorContext<
     return this.visitWithSpecificContext('visitChangePointCommand', context, input);
   }
 
-  // Expression visiting -------------------------------------------------------
+  public visitForkCommand(
+    parent: contexts.VisitorContext | null,
+    node: ESQLAstChangePointCommand,
+    input: types.VisitorInput<Methods, 'visitForkCommand'>
+  ): types.VisitorOutput<Methods, 'visitForkCommand'> {
+    const context = new contexts.ChangePointCommandVisitorContext(this, node, parent);
```

Comment:
Why `ChangePointCommandVisitorContext` here? 🤔  It should be `contexts.ForkCommandVisitorContext` no?

Reply:
Done in https://github.com/elastic/kibana/pull/216743/commits/21a070162a5ac704bf24899e331b88280c4a5304